app/models/dance.rb

```diff
@@ -14,7 +14,9 @@ class Dance < ActiveRecord::Base
     elsif user.admin?
       all
     else
-      where('publish= true OR user_id= ?', user.id)
```

Comment:
So this doesn't cut the cheese under SQLlite, but it does in postgres? I would not have expected that inconsistency.